api-docs/events.md

```diff
@@ -0,0 +1,45 @@
+# Events
+
+The list of events will document CU events.
+
+## Get single Event
+
+### Request
+
+```http
+GET /api/event/:id
+```
+
+### Response
+
+#### Success
+
+```json
+{
+  "id": 1,
+  "eventName": "Heath Fair",
+  "eventDate": "2019-09-16",
+  "eventLocation": "Saint Marks",
+  "programId": {
```

Comment:
Events aren't tied to a program in https://docs.google.com/spreadsheets/d/1wGx9pu2Udnn1MEWhas3u-PTe8uwZeStj20ItJ0UDirA/edit?usp=drive_web&ouid=117418984505592746114

Let's remove this since it's not needed. I don't want to build features for hypothetical futures.